MobileWallet/Common/Theme.swift

```diff
@@ -78,6 +78,11 @@ struct Colors: Loopable {
     // Splash
     let splashTitleColor = UIColor.white
 
+    //SplashCreatingWallet
+    let creatingWalletFirstLabel = UIColor.black
```

Comment:
Add a color resource for this and make it a universal color of black. We want to eventually support the ios 13 dark mode feature by just editing the color resources when we're ready.